search/base.py

```diff
@@ -4,23 +4,23 @@
 from unittest.mock import patch
 
 from django.test import (
-    override_settings,
     TestCase,
 )
 
-from search.indexing_api import recreate_index
+from search import indexing_api
+from search.indexing_api import recreate_index, delete_index
 
 
-class ESTestCase(TestCase):
+class ESTestCaseWithIndex(TestCase):
     """
     Set up ES index on setup
```

Comment:
can you change this description to something like: "Test class for test cases that need a live ES index"